config/manager/deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: TUF_ROOT
+            value: "/tmp/.sigstore"
```

Comment:
how is this env var used? a comment explaining it might be helpful in the future.

Reply:
A writable `$TUF_ROOT` env is required to initialize `sigstore-tuf-root`. Otherwise it would cause TUF client initialization to panic.

Reply:
okay got it, could you add a comment above this explaining this? it may help others understand its purpose

Reply:
I've added a comment about this.